nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -195,8 +195,8 @@ in
       hostKeys = mkOption {
         type = types.listOf types.attrs;
         default =
-          [ { type = "rsa"; bits = 4096; path = "/etc/ssh/ssh_host_rsa_key"; }
-            { type = "ed25519"; path = "/etc/ssh/ssh_host_ed25519_key"; }
+          [ { type = "rsa"; bits = 4096; path = "/etc/ssh/ssh_host_rsa_key"; rounds = 100; openSSHFormat = true; }
+            { type = "ed25519"; path = "/etc/ssh/ssh_host_ed25519_key"; rounds = 100; comment = "key comment"; }
```

Comment:
Do we need these default values or should they go to examples?

Reply:
Indeed, these were meant to be examples

Reply:
Are the parameter chosen by sshd unreasonable low?

Reply:
Not sure about that part, but the parameters are taken from [this](https://stribika.github.io/2015/01/04/secure-secure-shell.html) article, which is referenced in the file numerous times already.